tools/getkeys/package.json

```diff
@@ -14,11 +14,21 @@
   "main": "index.js",
   "scripts": {
     "start": "node ./index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "eslint": "eslint ./index.js",
+    "eslint:fix": "eslint ./index.js --fix",
+    "lint": "npm run eslint",
+    "lint:fix": "npm run eslint:fix"
   },
   "dependencies": {
     "@fluidframework/tool-utils": "^0.35.0",
     "azure-keyvault": "^3.0.4",
     "ms-rest-azure": "^2.6.0"
+  },
+  "devDependencies": {
+    "@fluidframework/eslint-config-fluid": "^1.0.0",
+    "@types/node": "^14.18.0",
+    "eslint": "~8.6.0",
+    "rimraf": "^2.6.2"
```

Comment:
It doesn't look like we're using this dependency. We can probably remove it.

Reply:
You are right. We actually don't need them. Removing here...